configuration.revanced.json

```diff
@@ -145,6 +145,46 @@
 					}
 				}
 			}
+		},
+		{
+			"includes": {
+				"channels": [952946952348270626, 953965039105232906],
+				"match": [
+					"(?i)(((tg|telegram|revanced|channel).*){2,})"
```

Comment:
This will likely never trigger. Common messages will certainly never include two words out of this group.

Reply:
works perfectly fine with "revanced telegram got hacked???"

Reply:
So basically just that or any combination of ReVanced and Telegram, which means stuff like "I downloaded from Telegram" will not trigger.

Reply:
hmmm

Reply:
fixed